site/scss/catalog/_shell.scss

```
@@ -107,16 +107,16 @@ body:has(.moo-catalog) {
-  --bs-btn-active-border-color: var(--bs-border-color);
-  min-height: 2rem;
+  --bs-btn-color: var(--bs-body-bg);
+  --bs-btn-bg: color-mix(in srgb, var(--bs-body-color) 88%, var(--bs-body-bg));
+  --bs-btn-border-color: var(--bs-btn-bg);
+  --bs-btn-hover-color: var(--bs-body-bg);
+  --bs-btn-hover-bg: var(--bs-body-color);
+  --bs-btn-hover-border-color: var(--bs-body-color);
+  --bs-btn-active-color: var(--bs-body-bg);
+  --bs-btn-active-bg: var(--bs-body-color);
+  --bs-btn-active-border-color: var(--bs-body-color);
+  min-height: var(--moo-catalog-control-height);
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)